src/doc/4.1/_sources/operating/backups.rst.txt

```diff
@@ -378,7 +381,19 @@ To take snapshots of multiple tables that are in different keyspaces the command
  Requested creating snapshot(s) for [catalogkeyspace.journal,cqlkeyspace.t] with snapshot 
  name [multi-ks] and options {skipFlush=false}
  Snapshot directory: multi-ks
- 
+
+Providing TTL for created snapshot
```

Comment:
Call Section `Expiring snapshots` and add it to its own section after `Listing Snapshots`. This will be important to add other features in the future like snapshot cleanup pause and resume.